base_import_security_group/README.rst

```diff
@@ -0,0 +1,104 @@
+===================
+Optional CSV import
+===================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:fe2d7959aa71ca276bb4b9fb395d9588490642e013a064a6c332860fa24edfeb
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fserver--ux-lightgray.png?logo=github
+    :target: https://github.com/OCA/server-ux/tree/18.0/base_import_security_group
+    :alt: OCA/server-ux
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/server-ux-18-0/server-ux-18-0-base_import_security_group
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/server-ux&target_branch=18.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module makes importing data from CSV and Excel files optional for
+each user, depending on whether Import CSV/Excel files is ticked on the
+Access Rights tab on the user form. This corresponds to a user group by
+the same name. Only users that belong to this group will have the Import
+records button available under the Favorites button of each list or
```

Comment:
```suggestion
records button available under the Settings icon of each list or
```

Reply:
Good catch! But maybe it should be replaced with 'Actions icon', right?
![image](https://github.com/user-attachments/assets/cf44e038-8d4b-4605-9c7d-ce196e3b40f3)

Reply:
Done now